pkg/apis/featurestates/v1alpha1/types.go

```diff
@@ -0,0 +1,41 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
The naming convention for this file does not match other CNS operator APIs. Not sure if there are any other restrictions you want to call out.

Reply:
Do you mean the name of the type.go should be apiname_types.go? I guess there are no such convention, for migration API we used types.go - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/apis/migration/v1alpha1/types.go